### PR TITLE
feat(pipelines) : change in input required modal to check jenkins status

### PR DESCRIPTION
**Pull request by invincibleJai**
### Pipelines

change in input required modal to check if jenkins is running, enable promote button only if jenkins is running

Tracks : 
https://github.com/openshiftio/openshift.io/issues/3356#issuecomment-396562994

### CODE REVIEW

**Comment by invincibleJai** [comment]
[test]

**Comment by joshuawilson** [comment]
The env var is not in place. Is there anything else you need to do to finish this?

File: src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.html [hunk 1]
```diff
@@ -4,18 +4,31 @@
   </modal-header>
 
   <modal-content>
-    <div class="input-action-message-panel">
-      <div *ngFor="let message of messageLines" class="input-action-message">
-        <p>{{message}}</p>
+    <ng-container *ngIf="this.jenkinsStatus; else jenkinsIdeled">
```
Comment (arunkumars08) [inline src/a-runtime-console/kubernetes/ui/pipeline/input-action-dialog/input-action-dialog.component.html]: I think there is a typo, not a problem though but good to be changed,
jenkinsIdeled to jenkinsIdled

**Comment by invincibleJai** [comment]
> The env var is not in place. Is there anything else you need to do to finish this?

@joshuawilson Env is needed , as other PRs are merged I feel env should be in place now.

**Comment by joshuawilson** [comment]
oops, that was a typo.

I meant to say "the env var ARE in place". I had just merged them, hence why I was asking.

**Comment by invincibleJai** [comment]
Thanks @joshuawilson  for help, this PR is open for review once all looks good can be merged.

**Comment by joshuawilson** [comment]
Is it not a work in progress anymore? Please update the title.

**Comment by invincibleJai** [comment]
@arunkumars08 have made the changes , PTAL

**Comment by arunkumars08** [comment]
@joshuawilson @invincibleJai Tested locally. Works fine as expected.

**Comment by invincibleJai** [comment]
thanks @arunkumars08 for review, @mindreeper2420 can you PTAL

**Pull request merged**